app/streaming/audio/renderers/sdlaud.cpp

```diff
@@ -73,17 +113,26 @@ bool SdlAudioRenderer::prepareForPlayback(const OPUS_MULTISTREAM_CONFIGURATION*
 
     // Start playback
     SDL_PauseAudioDevice(m_AudioDevice, 0);
+#endif
 
     return true;
 }
 
 SdlAudioRenderer::~SdlAudioRenderer()
 {
-    if (m_AudioDevice != 0) {
+#if SDL_VERSION_ATLEAST(3, 0, 0)
+    if (m_AudioStream) {
+        // Stop and destroy audio stream
+        SDL_PauseAudioDevice(SDL_GetAudioStreamDevice(m_AudioStream));
```

Comment:
These can be [SDL_PauseAudioStreamDevice](https://wiki.libsdl.org/SDL3/SDL_PauseAudioStreamDevice) to cut out the SDL_GetAudioStreamDevice call. There's an equivalent [SDL_ResumeAudioStreamDevice](https://wiki.libsdl.org/SDL3/SDL_ResumeAudioStreamDevice), too, for elsewhere.

While one _does_ need to resume to get a device from SDL_OpenAudioDeviceStream moving, pausing it before destroying the stream isn't necessary; it will halt playback just the same.